README.md

```diff
@@ -3925,6 +3943,7 @@ Message Type: **response**
 |`emergencyEvent`|VehicleDataResult|False|Information related to an emergency event (and if it occurred)|
 |`clusterModes`|VehicleDataResult|False|The status modes of the cluster|
 |`myKey`|VehicleDataResult|False|Information related to the MyKey feature|
+|`windowStatus`|Boolean|False|Information related to window status|
```

Comment:
@IGapchuk Checge to VD Result instead of boolean

Reply:
@IGapchuk fixed in 65b609a